Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "starship"
-version = "0.4.0"
+version = "0.3.2"
```

Comment:
Whoops! Looks like the version in `Cargo.toml` and `Cargo.lock` got changed.

Reply:
Ah, sorry. I had a bad feeling about that change while I was rebasing.